Library/Homebrew/brew.sh

```diff
@@ -506,11 +506,11 @@ esac
 # and also update references in docs/Installation.md,
 # https://github.com/Homebrew/install/blob/HEAD/install.sh and
 # MacOSVersion::SYMBOLS
-HOMEBREW_MACOS_NEWEST_UNSUPPORTED="14"
+HOMEBREW_MACOS_NEWEST_UNSUPPORTED="15"
 # TODO: bump version when new macOS is released and also update
 # references in docs/Installation.md and
 # https://github.com/Homebrew/install/blob/HEAD/install.sh
```

Comment:
Can you update this too?

Reply:
Install PR is at https://github.com/Homebrew/install/pull/807